### DISCUSSION TITLE
anyone know of a good script for checking streams?

### DISCUSSION THREAD

**Discussion by bitsbb01** [bitsbb01/m3u8_creator#4] [Q&A]
so been trying a few different scripts to check the m3u8 links so it'll remove the non-working one's.
and just cannot get it to work..

python3 ./scripts/streamcleaner.py --timeout 10.0 --input-file "./CompiledallStreams.m3u" --output-file "CompiledallStreams._checked.m3u"

which is meant to check the streams.. but nope, it just keeps failing..

**Comment by BellezaEmporium** [comment]
You could try to analyze both stream and server status code.

Use the requests library for it, it should be easy !
